src/components/RenderCanvas.vue

```diff
@@ -486,12 +529,14 @@ async function execFrame(timeMS: number, currentDisplayMode: ShaderType, playgro
 
     const timeElapsed = performance.now() - startTime;
 
+    frameID.value++;
     // Update performance info.
     timeAggregate += timeElapsed;
     frameCount++;
     if (frameCount == 20) {
-        let avgTime = (timeAggregate / frameCount);
+        let avgTime = timeAggregate / frameCount;
         frameTime.value = avgTime;
+        fps.value = Math.round(1000 / avgTime);
```

Comment:
It might be best to remove the frame time display since the fps counter now does its job.